docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst

```diff
@@ -100,6 +100,7 @@ There is one randomized controlled trial on Isatuximab treatment that is current
 2.0 Modeling aims and objectives
 ++++++++++++++++++++++++++++++++
 
+The obejective of this project is to provide Sanofi with a simulation design and implementation solution for a MM clinical registry, we will simulate outcomes rates among different populations to aid in registry design. For a population level of scope, we will assess the impact of Isatuximab in the populaiton of intertes at the completion of simulation. Estimating the incidence, prevalence, and survival of MM to be overall in the US, and in key sub-populationss listed above, under a business-as-usual treatment scenario and an Isatuximab treatment sceanrio over 5 years. The outcomes will be stratified by age, sex, and relavant demographic. 
```

Comment:
This looks great! There are a few typos/spelling updates to make, when next possible.

Reply:
Typo fixed